packages/app-project/src/components/ProjectHeader/ProjectHeaderContainer.spec.js

```diff
@@ -27,4 +39,19 @@ describe('Component > ProjectHeaderContainer', function () {
   it('should pass down the project title', function () {
     expect(componentWrapper.prop('title')).to.equal(PROJECT.display_name)
   })
+
+  it('should pass down the project homepage href if on another page', function () {
+    expect(componentWrapper.prop('href')).to.equal(`/projects/${OWNER}/${PROJECT_SLUG}`)
```

Comment:
At first read this was confusing because of what project slug means from the API, `:owner/:display_name`... so same comment about renaming the variable here.